### PR TITLE
Translate new strings to PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -669,39 +669,39 @@
...
-          "title": "ETH checksum instead of RSK checksum",
-          "description": "use lower case addresses as receiver for manual transactions"
+          "title": "ETH checksum ao invés do RSK checksum",
+          "description": "use endereços em letras minúsculas para receber transações"
         },
         "price": {
-          "title": "Shows price as ETH instead of (r)BTC",
-          "description": "currently we suggest checking your token balances in the SOVRYN My Wallet section."
+          "title": "É mostrado o preço como se fosse ETH em vez de (r)BTC ",
+          "description": "atulamente sugerimos você verificar o saldo dos seus ativos na seção minha carteira da Sovryn."
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:704]:
```suggestion
          "description": "atulamente sugerimos você verificar o saldo dos seus ativos na aba minha carteira da Sovryn."
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -669,39 +669,39 @@
...
         "checksum": {
-          "title": "ETH checksum instead of RSK checksum",
-          "description": "use lower case addresses as receiver for manual transactions"
+          "title": "ETH checksum ao invés do RSK checksum",
+          "description": "use endereços em letras minúsculas para receber transações"
         },
         "price": {
-          "title": "Shows price as ETH instead of (r)BTC",
-          "description": "currently we suggest checking your token balances in the SOVRYN My Wallet section."
+          "title": "É mostrado o preço como se fosse ETH em vez de (r)BTC ",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:703]:
```suggestion
          "title": "É mostrado o preço como se fosse ETH em vez de (r)BTC",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -669,39 +669,39 @@
...
-      "p2_4": "Nifty Wallet",
-      "p2_5": ". If you wish to continue with MetaMask please be aware of the following known errors:",
-      "knownErrors": {
+      "p1": "Verificamos que você tem a MetaMask instalada.",
+      "p2_1": "Existem problemas conhecidos ao usar a MetaMask com a SOVRYN. Sugerimos usar uma carteira de navegador como a ",
+      "p2_2": "Carteira Liquality",
+      "p2_3": " ou a ",
+      "p2_4": "Carteira Nifty",
+      "p2_5": ". Se você pretende continuar usando a MetaMask, esteja ciente do seguintes problemas:",
+      "knownErrors":
```
Comment (creed-victor) [inline src/locales/pt_br/translation.json:693]:
```suggestion
      "knownErrors": {
```

**Review by creed-victor — CHANGES_REQUESTED** [review]
Approve my suggestion which fixes json typo and prevents build to succeed

**Review by creed-victor — APPROVED** [review]

**Pull request merged**